### PR TITLE
PAR-1112: Updated views to not show soft deleted entities.

### PR DESCRIPTION
**Pull request by kalpaitch**
This is a temporary measure while we contemplate the implications of removing all these entities permanently.

### CODE REVIEW

File: sync/views.view.par_user_enforcement_list.yml [hunk 1]
```diff
@@ -935,6 +973,44 @@ display:
...
+            widget: select
+            multiple: false
+            remember: false
+            default_group: All
+            default_group_multiple: {  }
+            group_items: {  }
+          entity_type: par_data_enforcement_notice
+          entity_field: deleted
+          plugin_id: boolean
       filter_groups:
```
Comment (jon-edery) [inline sync/views.view.par_user_enforcement_list.yml]: Looks like the same filter has been added 2 times here.

Comment (jon-edery) [reply sync/views.view.par_user_enforcement_list.yml]: This maybe just a result of multiple displays on the view?

**Comment by jon-edery** [comment]
Looks gd.  would just confirm we are not adding the same filter 2 times on a the view and we are happy for the vbo export config to be added here.

File: sync/views.view.par_user_enforcement_list.yml (hunk 1 above)
Comment (kalpaitch) [reply sync/views.view.par_user_enforcement_list.yml]: Yea multiple displays. There is some reason whey the two displays had to use slightly different fields and therefore add the deleted field twice.

File: composer.json [hunk 2]
```diff
@@ -49,6 +49,7 @@
         "drupal/twig_field_value": "~1.1.0",
         "drupal/twig_tweak": "~1.8.0",
         "drupal/ultimate_cron": "^2.0@alpha",
+        "drupal/vbo_export": "^2.0",
```
[the remark this answers is not in the document]
Comment (kalpaitch) [reply composer.json]: Kinda trying to sneak it in yea :)

**Pull request merged**